### ISSUE TITLE
Select [only] mutations in lollipop

### ISSUE THREAD

**Issue by inodb** [cBioPortal/cbioportal#9560]
Currently the selections on mutation mapper behave like a checkbox. It would be good to have a select [only] option on hover:

![Screen Shot 2022-05-20 at 9 48 27 AM](https://user-images.githubusercontent.com/1334004/169542194-1b55060e-4cf6-4a37-9d8e-99f0e69f860f.png)

so e.g. select SV/Fusion [only]

**Comment by jjgao** [comment]
Good idea. Good Flights has something similar to that. Wondering if we have space to show the [only] option?

![image](https://user-images.githubusercontent.com/840895/169908918-0c7d7f67-0e43-4187-98f4-faa98a711edc.png)
